src/components/carousel/carousel.spec.tsx

```diff
@@ -26,6 +29,27 @@ function renderCarousel(props: CarouselProps, renderer = mount) {
 }
 
 describe("Carousel", () => {
+  describe("Deprecation warning", () => {
+    it("when user uses the component, a deprecation warning is raised only once in the console", () => {
+      const loggerSpy = jest.spyOn(Logger, "deprecate");
+      jest.restoreAllMocks();
```

Comment:
**suggestion**: we don't need to restore the mocks created with `spyOn` here. Restoring them at the end of the test should be sufficient:

```tsx
const loggerSpy = jest.spyOn(Logger, "deprecate");

/* mount and expects... */

loggerSpy.mockRestore()
```